Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -429,45 +537,98 @@ public string keyboardLayout
+        /// </summary>
+        /// <value>Control representing the period key.</value>
         public KeyControl periodKey => this[Key.Period];
+
+        /// <summary>
+        /// The '/' key. The key immediately to the left of <see cref="rightShiftKey"/>.
+        /// </summary>
+        /// <value>Control representing the forward slash key.</value>
         public KeyControl slashKey => this[Key.Slash];
+
+        /// <summary>
+        /// The '\' key. The key immediately to the right of <see cref="rightBracketKey"/> and
+        /// next to or above <see cref="enterKey"/>.
+        /// </summary>
+        /// <value>Control representing the backslash key.</value>
         public KeyControl backslashKey => this[Key.Backslash];
+
+        /// <summary>
+        /// The '[' key. The key immediately to the left of <see cref="rightBracketKey"/>.
+        /// </summary>
+        /// <value>Control representing the left bracket key.</value>
         public KeyControl leftBracketKey => this[Key.LeftBracket];
+
+        /// <summary>
+        /// The ']' key. The key in-between <see cref="leftBracketKey"/> to the left and
+        /// <see cref="backslashKey"/> to the right.
+        /// </summary>
+        /// <value>Control representing the right bracket key.</value>
         public KeyControl rightBracketKey => this[Key.RightBracket];
+
+        /// <summary>
+        /// The '-' key. The second key to the left of <see cref="backspaceKey"/>.
+        /// </summary>
+        /// <value>Control representing the minus key.</value>
         public KeyControl minusKey => this[Key.Minus];
 
         /// <summary>
-        /// The '=' key in the main key block. Key immediately to the left of the backspace key.
+        /// The '=' key in the main key block. The key in-between <see cref="minusKey"/> to the left
+        /// and <see cref="backspaceKey"/> to the right.
         /// </summary>
+        /// <value>Control representing the equals key.</value>
         public KeyControl equalsKey => this[Key.Equals];
 
         /// <summary>
-        /// The 'a' key. Key immediately to the right of the caps lock key.
+        /// The 'a' key. The key immediately to the right of <see cref="capsLockKey"/>.
         /// </summary>
+        /// <value>Control representing the a key.</value>
         public KeyControl aKey => this[Key.A];
+
+        /// <summary>
+        /// The 'b' key. The key in-between the <see cref="vKey"/> to the left and the <see cref="nKey"/>
+        /// to the right in the bottom-most row of alphabetic characters.
+        /// </summary>
+        /// <value>Control representing the b key.</value>
         public KeyControl bKey => this[Key.B];
+
+        /// <summary>
+        /// The 'c' key. The key in-between the <see cref="xKey"/> to the left and the <see cref="vKey"/>
+        /// to the right in the bottom-most row of alphabetic characters.
+        /// </summary>
+        /// <value>Control representing the c key.</value>
```

Comment:
So we document all letters up to C? :) (Would probably use some grep to put some standard phrase for all of them)

Reply:
Haha, no. Also incomplete. Locally have it about 75% complete and going. For reviewing, my approach probably sucks. Sorry.